backend/src/main/kotlin/fr/gouv/cacem/monitorenv/infrastructure/api/adapters/bff/outputs/MissionDetachedReportingDataOutput.kt

```diff
@@ -0,0 +1,29 @@
+package fr.gouv.cacem.monitorenv.infrastructure.api.adapters.bff.outputs
+
+import fr.gouv.cacem.monitorenv.domain.use_cases.reportings.dtos.ReportingDTO
+import java.time.ZonedDateTime
+
+data class MissionDetachedReportingDataOutput(
+    val id: Int,
+    val reportingId: Long,
+    val attachedToMissionAtUtc: ZonedDateTime? = null,
+    val detachedFromMissionAtUtc: ZonedDateTime? = null,
+) {
+    companion object {
+        fun fromReportingDTO(dto: ReportingDTO): MissionDetachedReportingDataOutput {
```

Comment:
Pourquoi on utilise le DTO ici ? On n'a besoin que du reporting

```suggestion
        fun fromReporting(reporting: Reporting): MissionDetachedReportingDataOutput {
```